core/drivers/crypto/caam/cipher/local.h

```diff
@@ -56,12 +73,12 @@ struct cipherdata {
  * @encrypt  Encrypt or decrypt direction
  * @src      Source data to encrypt/decrypt
  * @dst      [out] Destination data encrypted/decrypted
- * @blockbuf Saved block during previous streaming update
+ * @blocks   Additionnal data block to handle (input/output)
  */
 enum caam_status caam_cipher_block(struct cipherdata *ctx, bool savectx,
 				   uint8_t keyid, bool encrypt,
 				   struct caambuf *src, struct caambuf *dst,
-				   bool blockbuf);
+				   unsigned int blocks);
```

Comment:
Would it be possible to use an enum instead? If not, at least comment which defines are supposed to be used.

Reply:
Added ```enum caam_cipher_block```